code/missionui/missionshipchoice.h

```diff
@@ -86,7 +86,7 @@ void draw_wing_block(int wb_num, int hot_slot, int selected_slot, int class_sele
 void ship_select_init();
 void ship_select_do(float frametime);
 void ship_select_close();
-void ship_select_common_init();
+void ship_select_common_init(bool API_Access);
```

Comment:
It's dangerous to re-declare functions with default parameters.
I'd just throw this declaration out, since missionshipchoice.cpp includes missionscreencommon.h anyways

Reply:
Done